index.js

```diff
@@ -32,6 +33,20 @@ module.exports = {
   init(ssb, config) {
     ssb.db.registerIndex(IndexPlugin)
 
+    /**
+     * Determine how many bytes a feed is using on the log, and (on average)
+     * how many bytes it is using for indexes, and add them up.
+     * @param {number} logBytes
+     * @param {*} info
+     * @returns {number}
+     */
+    function sumLogAndIndexes(logBytes, info) {
```

Comment:
Naming is hard and it's late here and I have a headache. Probably could have found a better name for this in a better moment.